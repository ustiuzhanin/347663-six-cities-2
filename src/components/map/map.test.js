import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import RenderMap from './map.jsx';

it(`RenderMap's snapshot`, () => {
  const renderer = new ShallowRenderer();
  renderer.render(
      <RenderMap
        activeCity={{
          coords: [52.38333, 4.9],
          offers: [{location: [52.369553943508, 4.85309666406198]}]
        }}
      />
  );

  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
