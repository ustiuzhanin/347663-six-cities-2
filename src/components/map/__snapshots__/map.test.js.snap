// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderMap's snapshot 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
